roles/matrix-bot-maubot/templates/systemd/matrix-maubot.service.j2

```diff
@@ -0,0 +1,38 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Maubot
+{% for service in matrix_bot_maubot_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_bot_maubot_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ matrix_systemd_unit_home_path }}"
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} kill matrix-maubot 2>/dev/null || true'
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} rm matrix-maubot 2>/dev/null || true'
+
+ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-maubot \
+			--log-driver=none \
+            -e UID={{ matrix_user_uid }} \
+            -e GID={{ matrix_user_gid }} \
+            -v {{ matrix_bot_maubot_data_path }}:{{ matrix_bot_maubot_container_data_dir }}:z \
+			{% for arg in matrix_bot_maubot_container_extra_arguments %}
+			{{ arg }} \
+			{% endfor %}
+			--network={{ matrix_docker_network }} \
+            -p {{ matrix_bot_maubot_port }}:29316 \
```

Comment:
Exposing this to the host should be a config option

Reply:
You mean the line   `-p {{ matrix_bot_maubot_port }}:29316 \` ?

Reply:
yeah

Reply:
Done